tests/testGaussianPsfFactory.py

```diff
@@ -50,14 +50,14 @@ def testApply(self):
                             naiveSize += 1
                     else:
                         naiveSize = fixedSize
-                    for minSize in (None, max(1, naiveSize-1), naiveSize, naiveSize + 2):
+                    for minSize in (max(1, naiveSize-1), naiveSize, naiveSize + 2):
```

Comment:
That's what is worrying me. Why did the code have it originally and how sure are we it's all irrelevant.